build_tools/docker/base/Dockerfile

```diff
@@ -50,8 +50,8 @@ WORKDIR /install-cmake
 # to get the /usr/share path.
 # See https://github.com/moby/moby/issues/41383
 ARG CMAKE_MAJOR_VERSION=3
-ARG CMAKE_MINOR_VERSION=16
-ARG CMAKE_PATCH_VERSION=3
+ARG CMAKE_MINOR_VERSION=22
```

Comment:
Ergh, missed this, but this should've been upgraded to the *minimum* version on the maximum. I'll send a follow-up to drop it back to 3.17

Reply:
ah this is in order to validate that we support the minimum version? I was thinking upgrading directly to 22 was simpler, but I didn't think about how to validate the minimum version. Sorry about that.